.gitlab/trigger_release.yml

```diff
@@ -21,7 +21,7 @@
     - source /root/.bashrc
     - export RELEASE_VERSION=$(inv agent.version --major-version 7 --url-safe --omnibus-format)-1
     - export GITLAB_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-agent.gitlab_pipelines_scheduler_token --with-decryption --query "Parameter.Value" --out text)
-    - 'inv pipeline.trigger-child-pipeline --project-name "DataDog/agent-release-management" --git-ref "main"
+    - 'inv pipeline.trigger-child-pipeline --project-name "DataDog/agent-release-management" --git-ref "ducolombier/agentless-scanner"
```

Comment:
Opening a thread so we can think about merging your agent-release-management branch before merging this PR